config/initializers/orcid_client.rb

```diff
@@ -0,0 +1,7 @@
+SulOrcidClient.configure(
+  client_id: Settings.ORCID.CLIENT_ID,
+  client_secret: Settings.ORCID.CLIENT_SECRET,
+  base_url: Settings.ORCID.BASE_URL,
+  base_public_url: Settings.ORCID.BASE_PUBLIC_URL,
+  base_auth_url: Settings.ORCID.BASE_AUTH_URL 
```

Comment:
We typically use all lowercase to name Settings variables, but I'm assuming this PR is piggybacking off of existing shared_configs (that is still uppercased)? In which case I leave it to you whether it's timely or not to make this tweak.

Reply:
Yeah, this is following the existing conventions in sul_pub and using existing configuration.